guides/common/modules/proc_migrating-to-newly-deployed-satellite.adoc

```diff
@@ -0,0 +1,57 @@
+[id="migrating-to-newly-formed-{ProjectServer}]_{context}"]
+= Migrating to a Newly Formed {ProjectServer}
+
+*Before you begin*
+
+* Install a minimal {RHEL} 8 instance on the target machine with the capacity to store backup files.
+* Do not install any {RHEL} 8 software groups or third-party applications.
+
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/index[Performing a standard {RHEL} installation].
+
+== Creating a backup of {ProjectServer} or {SmartProxyServer} on {RHEL} 7
+
+Before you perform a fresh installation of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 system, back up your {ProjectServer} or {SmartProxyServer} data on the {RHEL} 7 system by creating an offline backup.
+
+If you recently created an offline backup, you can perform an incremental backup to update the existing backup.
+
+*Procedure*
+
+* Perform an offline or incremental backup on the source machine:
+** To perform an offline backup, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html/administering_red_hat_satellite/backing-up-satellite-server-and-capsule-server#performing-a-full-backup-of-satellite-or-capsule_admin[Performing a Full Backup of {ProjectServer} or {SmartProxyServer}].
+** To perform an incremental backup, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html/administering_red_hat_satellite/backing-up-satellite-server-and-capsule-server#performing-an-incremental-backup_admin[Performing an Incremental Backup].
+* Transfer the backup to the target machine.
+
+== Performing a fresh install of {ProjectServer} or {SmartProxyServer} to migrate to {RHEL} 8
+
+The fresh install method has three steps:
+
+. Create a backup of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 7 system source machine.
+. Perform a fresh install of the {ProjectServer} or {SmartProxyServer} on a minimal {RHEL} 8 instance on the target machine.
+. Restore the backup on the {RHEL} 8 instance.
+
+While the backup is restoring, you can use the {RHEL} 7 system on the source machine to reduce downtime.
+However, any changes you make at this time on the {RHEL} 7 system are not in the backup.
+
+[sidebar]
+Do not perform an offline backup without Pulp content or an online backup.
```

Comment:
Do not perform an offline backup without first doing an online backup? What does this mean?